ui/app/AppLayouts/Wallet/controls/MaxSendButton.qml

```diff
@@ -0,0 +1,41 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+
+import AppLayouts.Wallet 1.0
+
+import utils 1.0
+
+StatusButton {
+    id: root
+
+    required property double value
+    required property string symbol
+    required property bool valid
+    property var formatCurrencyAmount: (amount, symbol) => { return "FIXME" }
+
+    readonly property double maxSafeValue: WalletUtils.prepareForMaxSend(value, symbol)
```

Comment:
Ideally this value should be provided from outside, then there is no need to make call to `WalletUtils.prepareForMaxSend(value, symbol)` internally.

Reply:
I'd really keep that call inside (as it always was); even that function is used only here and nowhere else